index.html

```diff
@@ -1820,24 +1820,24 @@ <h3>Lifecycle Details</h3>
 <a>verifiable credentials</a> from <a>holders</a> for its supported actions.
           </li>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+The <a>issuer</a> issues some <a>verifiable credential(s)</a> to the <a>holder</a>.
```

Comment:
The <a>issuer</a> issues one or more <a>verifiable credentials</a> to the <a>holder</a>.